design-patterns/README.md

```diff
@@ -0,0 +1,21 @@
+# Software Architecture Patterns
+
+A software architecture is a description of the subsystems and components of a software system and the relationships between them. Subsystems and components are typically specified in different views to show the relevant functional and non-functional properties of a software system. The software system is an artifact. It is the result of the software design activity.
```

Comment:
I don't think we need to include definitions in the README. I recommend deleting lines 1-15.

Reply:
Any particular reason why having a definition doesn't make sense. My intent to keep was not so much for the external user but for the internal Vendia users so they can keep the definition of design-patterns consistent among us. Other than that I don't have a strong preference on whether to keep it or delete it. Let's discuss it during our SA sync tomorrow.